docs/configuration.adoc

```diff
@@ -381,3 +382,178 @@ backup and restore] functionality should be relied on to provide safe
 durability. The standalone mode's backup and restore operations can
 instead be used for creating operational snapshots of a node's indexes
 for scaling purposes.
+
+[#config-metrics]
+== Using metrics
```

Comment:
Let's go for "Monitoring" as the subheading for now. Eventually we will probably want to promote it to the top-level and give "Metrics" its own subheading.